### PR TITLE
stormpath-cli 0.0.2 (new formula)

### PR DESCRIPTION
**Pull request by rdegges**
In this commit, I'm officially adding support for the Stormpath CLI to Homebrew:
https://github.com/stormpath/stormpath-cli

Stormpath is an API service that manages user accounts for application
developers, and the CLI tool lets developers work with the service in numerous
ways.

This is an officially supported CLI, and homebrew formula. I'm also the author
of the underlying libraries, and am happy to maintain them.

### CODE REVIEW

**Comment by rdegges** [comment]
I've tested this out several times, seems to be working well! If anyone has feedback for me, I'd absolutely love it ^^ This is my first Homebrew formula =)

File: Library/Formula/stormpath-cli.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+class StormpathCli < Formula
+  desc "Command-line interface for https://stormpath.com/ user management"
+  homepage "https://github.com/stormpath/stormpath-cli"
+  url "https://github.com/stormpath/stormpath-cli/archive/0.0.2.tar.gz"
+  sha256 "50e334cb48cd0f9c0abdf77c1aa577c1d900547d8be39155d2248b995c044b0e"
+  head "https://github.com/stormpath/stormpath-cli.git"
+
+  depends_on :python
```
Comment (mikemcquaid) [inline Library/Formula/stormpath-cli.rb]: `depends_on :python if MacOS.version <= :snow_leopard`

File: Library/Formula/stormpath-cli.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/stormpath", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/stormpath-cli.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/stormpath-cli.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  resource "isodate" do
+    url "https://pypi.python.org/packages/source/i/isodate/isodate-0.5.4.tar.gz"
+    sha256 "42105c41d037246dc1987e36d96f3752ffd5c0c24834dd12e4fdbe1e79544e31"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[docopt stormpath pyjwt oauthlib requests six python-dateutil pydispatcher isodate].each do |r|
```
Comment (mikemcquaid) [inline Library/Formula/stormpath-cli.rb]: Is this all of them? If so just use `resources.each do |r|`

File: Library/Formula/stormpath-cli.rb (hunk 1 above)
Comment (rdegges) [inline Library/Formula/stormpath-cli.rb]: Thanks! Just implemented this <3

File: Library/Formula/stormpath-cli.rb (hunk 3 above)
Comment (rdegges) [inline Library/Formula/stormpath-cli.rb]: Ok, cleaned this up too! Sorry about this, not super great with ruby =D

File: Library/Formula/stormpath-cli.rb (hunk 2 above)
Comment (rdegges) [inline Library/Formula/stormpath-cli.rb]: Question: would it be possible for me to add a configuration file to make this work? This CLI tool requires Stormpath API keys present in order to do anything other than output help / version info, so if I could use travis encrypt-file and modify the travis.yml script (not sure if that's something you want), then I could write a more extensive test case here.

The only downside atm is that this CLI doesn't do anything that doesn't require Stormpath authentication, so I can't test it out very much other than with the help output :(

Comment (mikemcquaid) [inline Library/Formula/stormpath-cli.rb]: Ok, that test is fine as-is then, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by rdegges** [comment]
Thanks so much! It is exciting to get my first formula into brew ^^

**Pull request closed without merging**